docs/configuration.md

```diff
@@ -109,6 +109,12 @@ redis:
 		maxidle: 16
 		maxactive: 64
 		idletimeout: 300s
+uploadpurging:
```

Comment:
http://www.thesaurus.com/browse/maintenance?s=t  lead me to 

ab·lu·tion
əˈblo͞oSHən/Submit
noun
the act of washing oneself (often used for humorously formal effect).
"the women performed their ablutions"
a ceremonial act of washing parts of the body or sacred containers.

I'll stick with maintenance.